docs/index.rst

```diff
@@ -30,51 +30,64 @@ Virtualenv
+Python applications will often use external libraries that are not part of the standard Python
+library. The applications will often require a specific version  of that library where a particular
+bug has been fixed. Imagine you have an application that needs version ``1.2.3`` of ``foo``,
+but another application requires version ``2.3.4``. Installing one version of the library would
+make the other application unable to run. So, If you install everything into your system Python
+it's easy to end up in a situation where two applications have conflicting requirements.
 
 Or more generally, what if you want to install an application *and leave it be*? If an application works, any change
 in its libraries or the versions of those libraries can break the application. Also, what if you can't install packages
-into the global ``site-packages`` directory, due to not having permissions to change the host python environment?
+into the global ``site-packages`` directory, due to not having permissions to change the system Python environment?
 
 In all these cases, ``virtualenv`` can help you. It creates an environment that has its own installation directories,
 that doesn't share libraries with other virtualenv environments (and optionally doesn't access the globally installed
 libraries either).
 
-Useful links
-------------
+Quick Start
+-----------
+The recommended way to install virtualenv is through pip: (for other methods see :doc:`Installation <installation>`)
 
-**Related projects, that build abstractions on top of virtualenv**
+.. code-block:: console
 
-* :pypi:`virtualenvwrapper` - a useful set of scripts for creating and deleting virtual environments
-* :pypi:`pew` - provides a set of commands to manage multiple virtual environments
-* :pypi:`tox` - a generic virtualenv management and test automation command line tool, driven by a ``tox.ini``
-  configuration file
-* :pypi:`nox` - a tool that automates testing in multiple Python environments, similar to tox,
-  driven by a ``noxfile.py`` configuration file
+    $ Python -m pip install --user virtualenv
+    $ virtualenv --version
 
-**Tutorials**
+To create a virtualenv using the default Python instance available on the system
 
-* `Corey Schafer tutorial <https://www.youtube.com/watch?v=N5vscPTWKOk>`_ on how to use it
-* `Using virtualenv with mod_wsgi <http://code.google.com/p/modwsgi/wiki/VirtualEnvironments>`_
+.. code-block:: console
 
-**Presenting how the package works from within**
+    $ virtualenv venv/
 
-* `Bernat Gabor: status quo of virtual environments <https://www.youtube.com/watch?v=o1Vue9CWRxU>`_
-* `Carl Meyer: Reverse-engineering Ian Bicking's brain: inside pip and virtualenv
-  <http://pyvideo.org/video/568/reverse-engineering-ian-bicking--39-s-brain--insi>`_
+To activate a virtualenv
+
+Bash
+^^^^
+
+.. code-block:: console
+
+    $ source venv/bin/activate
+
+Windows
+^^^^^^^
+
+.. code-block:: console
+
+    venv/bin/activate.bat
```

Comment:
Also no for this. Assumes user runs with command prompt, but that's not the case with git bash or PowerShell.

Reply:
I have removed the Quick Start section